src/main/java/com/sphy/hotelmanagementapplication/exception/ApiException.java

```diff
@@ -0,0 +1,33 @@
+package com.sphy.hotelmanagementapplication.exception;
+
+import org.springframework.http.HttpStatus;
+
+import java.time.ZonedDateTime;
+
+/***
+ * created by gp
+ */
+public class ApiException {
+
+    private final String message;
+    private final HttpStatus httpStatus;
+    private final ZonedDateTime zonedDateTime;
+
+    public ApiException(String message, HttpStatus httpStatus, ZonedDateTime zonedDateTime) {
+        this.message = message;
+        this.httpStatus = httpStatus;
+        this.zonedDateTime = zonedDateTime;
```

Comment:
This can be calculated in cosnstructor as get now in milliseconds, no need for a programmer to add it

Reply:
if we don't added it don't going to sow it on the error

Reply:
oh ok thanks